api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -577,6 +577,8 @@ message ScopedRoutes {
     }
 
     // The final scope key consists of the ordered union of these fragments.
+    // Fragments of the built scope key are compared in order with the fragments of a ScopedRouteConfiguration.
```

Comment:
I think this sentence is confusing given the context carried over from the previous sentence. The final scope key is referred to as "built", creating ambiguity (i.e., is it different than the "final" scope key referenced above?).

I suggest merging this sentence with the prior sentence; e.g.,

"The final (built) scope key consists of the ordered union of these fragments, which are compared in order with the fragments of a \<RST link\>ScopedRouteConfiguration".

Reply:
done